CODE_OF_CONDUCT.md

```diff
@@ -0,0 +1,5 @@
+All members of the Hubble community must abide by the [Cilium Community Code of
```

Comment:
```suggestion
All members of the Cilium community must abide by the [Cilium Community Code of
```

Reply:
🤦 also fixed the same copy&paste mistake in `CODE_OF_CONDUCT.md`. Thanks.